src/app.js

```diff
@@ -387,26 +387,52 @@ async function main() {
   });
 
   app.all("/leather/:type/:color", cors(), async (req, res) => {
-    let file;
+    const { type, color } = req.params;
 
-    if (!["boots", "leggings", "chestplate", "helmet"].includes(req.params.type)) {
+    if (!["boots", "leggings", "chestplate", "helmet"].includes(type)) {
       throw new Error("invalid armor type");
     }
 
-    const { type } = req.params;
+    if (!/^[0-9a-fA-Z]{6}$/.test(color)) {
```

Comment:
Shouldn't this be a-fA-F?

Reply:
72d00e0581adab835fb6a0c2d06e3e1382430d7f